src/nsm.h

```diff
@@ -512,6 +533,7 @@ NSM_EXPORT int _nsm_osc_announce_reply ( const char *path, const char *types, lo
 
     nsm->nsm_is_active = 1;
     nsm->_session_manager_name = strdup( &argv[2]->s );
+    nsm->_session_manager_features = strdup( &argv[3]->s );
```

Comment:
this value is not deallocated, leading to a memory leak.

Reply:
Nice cached. Fixed now.